src/json/townRoles.json

```diff
@@ -26,7 +26,7 @@
         "alignment": "Town (Townie)",
         "description": "The traitor is a member of the citizen team, if she is attacked by the mafia team, he is not killed, and from the next night of the game, he wakes up with the mafia members by betraying the town team and becomes a mafia.",
         "img": "../assets/images/roles/Traitor.png",
-        "href": "/townRoles/Traitor"
+        "href": "/roles/townRoles/Traitor"
```

Comment:
where do you need href?

Reply:
you can simply generate this attribute only using `name` attribute and not repeating it in all cases. As a rule of thumb: As a programmer, when you are doing the same thing for more than 1 time, means if something is being repeated +2 times, there is 99% chance that it can be centralized, in other words, you can use a technic to loop through, generate something, automate it, reuse things etc. this is one of the examples. it's not scalable! it's not reusable, it's bug-prune and etc.
I ask you a question: tomorrow you want to rename` /roles/` to` /gameRoles/` or `/townRoles/` to `/characterRoles/ `what happens? I give you the answer: you have to change it in all places! again and again...

but if you have a function like this:

```
function getHref(cardName) {
     return '/roles/townRoles/${cardName}';
}
```

if anything changes, you just need to change it in one place. see how it can be centralized?